modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/main.es.js

```diff
@@ -72,18 +71,15 @@ class Form extends Component {
 		const dependencies = [
 			this._createEditor('nameEditor').then((editor) => {
 				this._eventHandler.add(
-					dom.on(
-						editor.element.$,
+					editor.element.$.addEventListener(
```

Comment:
Why are we adding some listeners to the `eventHandler` (so it takes care of removing all of them when calling to `removeAllListeners` on dispose) and some others don't (so we need to manually remove them on dispose)?

Reply:
In this example i left `eventHandler` because of the baggage that comes with the new event listeners. I'd have to call `this._createEditor()` in order to remove those listeners. And since the task was to remove `on` and `once`, I took the easier road and did only what was asked in the task to not complicate everything for everyone involved.

Reply:
fair enough